src/index.ts

```diff
@@ -449,16 +449,31 @@ class Envoy {
         if (!appInfo) {
           log.error("Conductor call 'appInfo' returned non-success response: %s", appInfo);
           throw new HoloError(`Failed to call 'appInfo' for installed_app_id'${installed_app_id}'.`);
+        } else if (appInfo.status!.inactive) {
+          log.error("Conductor call 'appInfo' returned as inactive because: %s", JSON.stringify(appInfo.status!.inactive));
+          if (appInfo.status!.inactive!.reason!.quarantined) {
```

Comment:
Is there a reason to add these `!`s?

Reply:
In TS it acts as a non-null assertion operator and tells the compiler that that expression or obj cannot/will not be null or undefined here, which removes it from complaining about the possibility of it being null or undefined.  Overall, I just find if more explicit.
https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#non-null-assertion-operator